pkg/apis/kops/networking.go

```diff
@@ -37,22 +37,22 @@ type NetworkingSpec struct {
 	GCE        *GCENetworkingSpec        `json:"gce,omitempty"`
 }
 
-// ClassicNetworkingSpec is the specification of classic networking mode, integrated into kubernetes
+// ClassicNetworkingSpec is the specification of classic networking mode, integrated into kubernetes.
+// Support been removed since kops 1.4.
 type ClassicNetworkingSpec struct {
 }
 
 // KubenetNetworkingSpec is the specification for kubenet networking, largely integrated but intended to replace classic
 type KubenetNetworkingSpec struct {
 }
 
-// ExternalNetworkingSpec is the specification for networking that is implemented by a Daemonset
-// It also uses kubenet
+// ExternalNetworkingSpec is the specification for networking that is implemented by a Daemonset.
+// Uses the kubenet networking provider.
```

Comment:
I think "networking provider" is misleading as it doesn't use the kops Kubenet provider. It uses the Kubernetes kubenet networking code.

Reply:
Amended.

Reply:
Resolved